### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andresrinivasan**
Add quick build instructions.

### CODE REVIEW

**Comment by jpwhitemn** [comment]
@andresrinivasan @cloudxxx8 - is this PR still relevant and can we approve and merge it?

**Comment by andresrinivasan** [comment]
I hope those couple of lines of instructions will save someone else the time to figure it out. Remember when we could just use `ant`?

**Comment by jpwhitemn** [comment]
Andre - you are missing DCO

**Comment by andresrinivasan** [comment]
Corrected

**Comment by cloudxxx8** [comment]
recheck

**Comment by cloudxxx8** [comment]
DCO still failed

File: README.md [hunk 1]
```diff
@@ -3,4 +3,16 @@ Main Author: Cloud Tsai
...
+
+## Prerequisites
+
+* Maven
+* Java 8
+
+## Building
+
+`mvn clean install`
+`java -jar target/device-virtual-0.5.0-SNAPSHOT.jar`
```
Comment (cloudxxx8) [inline README.md:16]: the latest version number is 0.7.0

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

**Comment by eb-oss** [comment]
Closing PR; project is being archived.

**Pull request closed without merging**